### ISSUE TITLE
Sponsor Logo Sizing

### ISSUE THREAD

**Issue by elliotwhitehead** [HackCU/LocalHackDay2016#12]
So I was going to work on resizing all of the sponsor logos per our discussion on Friday (make all the images 600px wide X 400px tall, but It looks like they already are. Did somebody already handle this, or did I misunderstand my objective from our meeting?

**Comment by AparaV** [comment]
Last year, we tried to maintain this. But, as we got busy with more sponsors, we got a little sloppy and either did not resize it or resized it badly.
I also think the CSS was a bit messed up - there were multiple CSS files with the same class/id each overriding the previous one. That's another reason why the sponsors section looks badly organized.

Don't worry too much about the past sponsors section. We will be getting rid of it once we get 2/3 sponsors. So it's not worth making it look great imo (there's no one stopping you if you really want to!). A better task would be finding all the remaining CSS conflicts and fixing them.

But we should try to follow this style moving forward.
